strategy-infura-refresh.go

```diff
@@ -283,6 +314,46 @@ func removeInstanceProtection(ctx context.Context, sess *session.Session, asgNam
 	return err
 }
 
+// suspendAutoscaling disable the autonomous scaling actions on an ASG
+// - scaling based on cloudwatch alarms
+// - scaling based on predictive features
+func suspendAutoscaling(ctx context.Context, sess *session.Session, asgName string) error {
```

Comment:
As I was mentioning elsewhere, I think the safest route is to at least also pause `AZRebalance` or you might end up in the same situation.